lib/liquid/tags/assign.rb

```diff
@@ -10,6 +10,10 @@ module Liquid
   class Assign < Tag
     Syntax = /(#{VariableSignature}+)\s*=\s*(.*)\s*/om
 
+    def self.syntax_error_translation_key
+      "errors.syntax.assign".freeze
```

Comment:
Why use a method for a constant?

Reply:
Ah this can be removed now that `local` is removed too – it allowed overriding the error name.

Reply:
This was marked resolved, but the method remains.